package.json

```diff
@@ -5,9 +5,27 @@
   "peerDependencies": {
     "react": "18.2.0",
     "react-native": "0.71.4",
-    "stream-chat": "^3.7.0",
     "react-native-android-keyboard-adjust": "^1.2.0",
+    "react-native-device-info": "^10.9.0",
+    "react-native-safe-area-context": "^3.4.1",
     "RNDownload": "git+https://github.com/railroadmedia/RNDownload.git#master-musora-app",
-    "react-native-device-info": "^10.9.0"
+    "stream-chat": "^3.7.0"
```

Comment:
might be worth updating to latest version, I looked into the methods and I don't think they changed syntax or method names